### PR TITLE
NXP-29734: Upgrade Kafka Chart to bitnami one

### PR DESCRIPTION
**Pull request by kevinleturc**

### CODE REVIEW

**Comment by kevinleturc** [comment]
The Confluent Kafka version doesn't work out of the box with the bitnami chart.

**Comment by kevinleturc** [comment]
_Zookeeper:3.6.0_ is not compatible with the current version of bitnami chart due to missing `nc` command into the image -> use _3.6.2_.

**Comment by kevinleturc** [comment]
https://github.com/nuxeo/nuxeo/pull/4415 has successfully tested the Bitnami Kafka chart.

**Review by troger — APPROVED** [review]

File: nuxeo/values-production.yaml [hunk 1]
```diff
@@ -42,9 +42,7 @@ elasticsearch:
 
 ## Values taken from the Kafka chart: https://github.com/helm/charts/blob/master/incubator/kafka/values.yaml
```
Comment (ataillefer) [inline nuxeo/values-production.yaml:43]: can you please update the chart source

File: nuxeo/values.yaml [hunk 2]
```diff
@@ -124,18 +124,23 @@ elasticsearch:
 ## Values overridden from the Kafka chart: https://github.com/helm/charts/blob/master/incubator/kafka/values.yaml
```
Comment (ataillefer) [inline nuxeo/values.yaml:124]: update the chart source

File: nuxeo/values.yaml [hunk 3]
```diff
@@ -124,18 +124,23 @@ elasticsearch:
...
-    enabled: false
-  prometheus:
-    operator:
+    persistence:
+      enabled: false
+  serviceAccount:
+    create: false
+  metrics:
+    kafka:
+      enabled: false
```
Comment (ataillefer) [inline nuxeo/values.yaml:142]: aren't they disabled by default?

File: nuxeo/values.yaml [hunk 4]
```diff
@@ -124,18 +124,23 @@ elasticsearch:
...
+    serviceMonitor: # prometheus - require metrics.kafka.enabled=true
       enabled: false
```
Comment (ataillefer) [inline nuxeo/values.yaml:144]: disabled by default no?

**Comment by ataillefer** [comment]
Also, can you please update the README to get rid of the incubator occurrences.

File: nuxeo/values.yaml (hunk 3 above)
Comment (kevinleturc) [reply nuxeo/values.yaml:142]: Indeed, will remove them.

Comment (kevinleturc) [reply nuxeo/values.yaml]: Removed

File: nuxeo/values.yaml (hunk 4 above)
Comment (kevinleturc) [reply nuxeo/values.yaml]: Removed

**Review by ataillefer — DISMISSED** [review]
:muscle:

**Review by troger — APPROVED** [review]

**Review by ataillefer — APPROVED** [review]

**Pull request merged**